docs/Deploy HotStack on  PSI.md

```diff
@@ -0,0 +1,273 @@
+```bash
+sudo dnf config-manager --enable crb
+sudo dnf install -y centos-release-openstack-dalmatian.noarch
+```
+
+```bash
+sudo dnf install -y python-openstackclient
+```
+
+**Heat Client:
+```
+sudo dnf install -y python-heatclient
+```
+
+## 2.3 Cloning the HotStack Repository
+
+
+```bash
+git clone https://github.com/openstack-k8s-operators/hotstack.git
+cd hotstack
+```
+## 2.4 OpenStack Service Prerequisities
+
+- **Glance (image service):
+	- **iPXE Image**: The `ocp_agent_installer`  role utilizes "PXE bootstrap-artifacts", meaning OpenShift Container Platform (OCP) instances will be network-booted.
+	- Document to build and upload ipxe image is referenced within the HotStack repository [here](https://github.com/openstack-k8s-operators/hotstack/tree/main/ipxe)
+
+  * **Controller Node Image**: A dedicated image for the "controller" node must also be available in Glance.
+	  * We need `dnsmasq` pre-installed to enable the DNS service on the controller to initailize without external package downloads.
+	  * Document to build upload controller image is documented within the HotStack repository [here](https://github.com/openstack-k8s-operators/hotstack/tree/main/images)
+
+- **Nova (Compute Service):**
+	- **Flavors**: HotStack scenarios expect specific Nova flavors to be available for the instances it deploys.
+	- We can create flavor using following command:
+```bash
+openstack flavor create hotstack.small --public --vcpus 1 --ram 2048 --disk 20
+openstack flavor create hotstack.medium --public --vcpus 2 --ram 4096 --disk 40
+openstack flavor create hotstack.large --public --vcpus 4 --ram 8192 --disk 80
+openstack flavor create hotstack.xlarge --public --vcpus 8 --ram 16384 --disk 160
+```
+
+* Creating flavor is privileged task in OpenStack, not permitted for regular users. It is recommended use the existing flavor that matches the above vpcus, ram, disk.
+* To use existing flavor, we have to edit bootstrap_vars.yml file (e.g. `scenarios/uni01alpha/bootstrap_vars.yml`)
+* Example of `bootstrap_vars.yml` snippet, with existing flavor
+```yaml
+  controller_params:
+    image: hotstack-controller
+    flavor: g.standard.xs
+  ocp_master_params:
+    image: ipxe-boot-usb
+    flavor: ocp4.single-node
+  compute_params:
+    image: CentOS-Stream-9-latest
+    flavor: g.standard.xl
+```
+
+* **Keystone (Identity service)**:
+	* **OpenStack Cloud Credentials (cloud-secret.yaml)**: HotStack requires access to the OpenStack API.
+	* This is managed via a credentials file , commonly named `cloud-secret.yaml`
+	* It can be created in two ways using OpenStack Horizon or CLI.
+	* Using Horizon (GUI): Click on Identity -> Create Application credentials -> Give name and secret of your choice -> tick unrestricted -> Create Application credentials -> download `cloud-secret.yaml`
+	* CLI command to create `cloud-secret.yaml`
+	```bash
+openstack application credential create --unrestricted hotstack-app-credential
```

Comment:
If you click on the 3 dots, and select view file - this example command for application credentials do not render correctly.